src/validation.cpp

```diff
@@ -1695,15 +1695,15 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptPackage(const Package& package,
         return PackageMempoolAcceptResult(package_state_quit_early, {});
     }
 
-    // IsChildWithParents() guarantees the package is > 1 transactions.
-    assert(package.size() > 1);
```

Comment:
I wouldn't remove this, but rather update to:

```
    // IsChildWithParents() guarantees the package is not empty.
    assert(!package.empty());
```

After all, if the package is empty, we'd have UB a few lines down.